### PR TITLE
Feature/dashboard

### PR DESCRIPTION
**Pull request by andela-ekupara**
@gottsohn This PR features changes I made to the Trime dashboard where I added more details to the one we had initially. Please review and merge.

### CODE REVIEW

File: app/scripts/components/Dashboard/Orgs.jsx [hunk 1]
```diff
@@ -0,0 +1,55 @@
+(function() {
+  'use strict';
+  var React = require('react');
+   var ProjectStore = require('../../stores/ProjectStore');
```
Comment (andela-ekupara) [inline app/scripts/components/Dashboard/Orgs.jsx]: Nitpick

Separate variables with a comma

File: app/scripts/components/Dashboard/ProjectsTable.jsx [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  var ProjectStore = require('../../stores/ProjectStore');
+  var ProjectActions = require('../../actions/ProjectActions');
+
+  var ProjectsTable = React.createClass({
+    render: function() {
+      var self = this;
+      var filteredProjects = self.props.projects.filter(function(project){
+        return project.org_id === self.props.orgId;
+      });
+      var projectRow = filteredProjects.map(function(project, index) {
```
Comment (andela-ekupara) [inline app/scripts/components/Dashboard/ProjectsTable.jsx]: Code Quality: Styling

More whitespace

File: app/scripts/components/Dashboard/OrgsModal.jsx [hunk 3]
```diff
@@ -0,0 +1,54 @@
...
+  };
+
+
+  var OrgsModal = React.createClass({
+
+    getInitialState: function() {
+      return { modalIsOpen: false };
+    },
+
+    openModal: function() {
```
Comment (andela-ekupara) [inline app/scripts/components/Dashboard/OrgsModal.jsx]: Nitpick

Use a toggle function

File: app/scripts/components/Dashboard/ProjectsTable.jsx (hunk 2 above)
Comment (gottsohn) [inline app/scripts/components/Dashboard/ProjectsTable.jsx]: Where exactly do you want to add the whitespace?

File: app/scripts/components/Dashboard/OrgsModal.jsx (hunk 3 above)
Comment (gottsohn) [inline app/scripts/components/Dashboard/OrgsModal.jsx]: @andela-ekupara 
Also in line _24_, the object's opening and closing curly brackets should be on different lines, using the common indentation.

**Pull request merged**